### PR TITLE
Include static HAL deps in platformio build path

### LINKED ISSUE #59: Integrate build with PlatformIO
The build currently depends on a Makefile. We should figure out and then update the configuration in `platformio.ini` to allow VSCode/PlatformIO to build the source code.

### PR DESCRIPTION
**Pull request by sdiemert**
Closes #59

### CODE REVIEW

**Comment by sdiemert** [comment]
@patchworquill @LaureMillet please review this change and approve to merge into master branch.

File: platformio.ini [hunk 1]
```diff
@@ -12,4 +12,4 @@
 platform = ststm32
 framework = stm32cube
 board = nucleo_f401re
-build_flags = -DF4
\ No newline at end of file
+build_flags = -DF4 -I lib/stm32/STM32F4xx_HAL/Inc -I lib\stm32\CMSIS\Device\ST\STM32F4xx\Include -I lib\stm32\CMSIS\Include
```
Comment (LaureMillet) [inline platformio.ini:15]: why are the adress using both` / `and `\ `?

Comment (sdiemert) [reply platformio.ini:15]: ugh...got catch. This explains an issue i had earlier. I'll amend.

Comment (sdiemert) [reply platformio.ini]: fixed in latest commit.

Comment (patchworquill) [reply platformio.ini]: That explains it!

**Review by patchworquill — APPROVED** [review]
Fixes previous bug where `platformio.ini` was missing / separators between dirs

**Pull request merged**